src/api/app/views/webui2/webui/project/_project_packages.html.haml

```diff
@@ -21,10 +21,14 @@
       .pt-4
         %ul.list-inline
           %li.list-inline-item
-            = link_to(project_new_package_path(project: project)) do
+            %a{ href: '#', data: { toggle: 'modal', target: '#new-package-modal' } }
               %i.fas.fa-plus-circle.text-primary
               Create package
```

Comment:
To be consistent with the other links on the project show page, we should capitalize these 2 links too. So it becomes then 'Create **P**ackage' and 'Branch **E**xisting **P**ackage'

Reply:
@dmarcoux done